instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py

```diff
@@ -503,6 +531,14 @@ def _instrumented_lambda_handler_call(  # noqa pylint: disable=too-many-branches
                 kinesisTriggerSpan = tracer.start_span(span_name, context=parent_context, kind=SpanKind.CONSUMER, links=links)
                 kinesisTriggerSpan.set_attribute(SpanAttributes.FAAS_TRIGGER, "pubsub")
                 kinesisTriggerSpan.set_attribute("faas.trigger.type", "Kinesis")
+                kinesisTriggerSpan.set_attribute(SpanAttributes.MESSAGING_SYSTEM, "aws.kinesis")
+                kinesisTriggerSpan.set_attribute(SpanAttributes.MESSAGING_URL, queue_url)
+
+                try:
+                    kinesisTriggerSpan.set_attribute(SpanAttributes.MESSAGING_DESTINATION, queue_url.split("/")[-1])
```

Comment:
Do we have example for this URL vs in other event sources? I see here we split by the URL by `/`, in the previous event it's by `:`. With example it would be easier to follow, as I don't know how the URLs look from the top of my head.